common/fs/src/cache/mod.rs

```diff
@@ -1700,6 +1700,7 @@ mod tests {
     }
 
     /// Creates a symlink directory
+    #[cfg(unix)]
```

Comment:
Nit: put unix tag underneath test tag for consistency.

Reply:
this is temp, all tests must be fixed and all these ifdefs removed.